src/Resources/views/autocomplete/javascript.html.twig

```diff
@@ -0,0 +1,125 @@
+{% if ssm_autocomplete_enabled() %}
```

Comment:
My idea for refactoring this file is to divide it into multiple files, so that it's easier for the user to pick and choose and override smaller parts.

Wdyt?